### ISSUE TITLE
escuelafrontend: Inaccessible color contrast in header

### ISSUE THREAD

**Issue by zacjones93** [skillrecordings/products#213]
I noticed that the color contrast on the escuela frontend header dips into inaccessible ranges. 

Because of the gradient, the color contrast goes from `2.17` (fail) to `3.09` (AA+) at it's darkest. 

Ideally we don't want to go below `4.5` which would be darkening the gradient to `#188390` but I'm not sure how that would make the overall colors look.

![Escuela Frontend color contrast](https://user-images.githubusercontent.com/6188161/162785946-947195b7-9864-4af2-abe7-23b268d7d13e.jpg)
